docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -155,13 +179,62 @@ repository storage then you are responsible for its reliability.
 
 experimental::[]
 
-By default a {search-snap} copies the whole snapshot into the local cluster as
-described above. You can also configure a shared snapshot cache which is used
-to hold a copy of just the frequently-accessed parts of shards of indices which
-are mounted with `?storage=shared_cache`. If you configure a node to have a
-shared cache then that node will reserve space for the cache when it starts up.
+To mount a shared cache of a snapshot, you must use the the
+`xpack.searchable.snapshot.shared_cache.size` setting to reserve space for the
+cache on one or more nodes. Indices mounted as a shared cache can only be
+allocated to nodes that have this setting explicitly configured.
 
 `xpack.searchable.snapshot.shared_cache.size`::
 (<<static-cluster-setting,Static>>, <<byte-units,byte value>>)
 The size of the space reserved for the shared cache. Defaults to `0b`, meaning
 that the node has no shared cache.
+
+For example:
+
+[source,yaml]
+----
+xpack.searchable.snapshot.shared_cache.size: "4TB"
+----
+
+IMPORTANT: Currently, you can configure
+`xpack.searchable.snapshot.shared_cache.size` on any node. In a future release,
+you will only be able to configure this setting on nodes with the
+<<data-frozen-node,frozen data>> role.
+
+You can set `xpack.searchable.snapshot.shared_cache.size` to any size between a
+couple of gigabytes up to 90% of available disk space. We only recommend higher
+sizes you use the node exclusively as a frozen tier or for searchable snapshots.
+
+[discrete]
+[[searchable-snapshots-frozen-tier-on-cloud]]
+==== Configure a frozen tier on the {ess}
+
+The frozen data tier is not yet available on the {ess-trial}[{ess}]. However,
+you can configure another tier to use <<shared-cache,shared snapshot caches>>.
+This effectively recreates a frozen tier in your {ess} deployment. Follow these
+steps:
+
+. Choose an existing tier to use. Typically, you'll use the cold tier, but the
+hot and warm tiers are also supported. You can use this tier as a shared tier, or
+you can dedicate the tier exclusively to the shared snapshot cache.
```

Comment:
I felt that we should give some kind of recommendation here. Using the cold tier made sense to me, but I wasn't sure how well this would play with ILM defaults.